Sources/Classes/Timing Functions/DurationTimingFunction.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DurationTimingFunction.swift
+//  Spruce
+//
+//  Created by Jackson Taylor on 3/20/17.
+//  Copyright © 2017 WillowTree Apps, Inc. All rights reserved.
+//
+
+import Foundation
+
+struct DurationTimingFunction: TimingFunction {
+    var duration: TimeInterval = 0.3
+    
+    public init() {
+        
+    }
+    
+    public init(duration: TimeInterval) {
```

Comment:
Rather than the default on the stored property you could consider a default value for this parameter and remove the empty `init` above.